src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -115,6 +115,8 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(CassandraStorage.class);
 
+  private static boolean firstAttempt = true;
```

Comment:
is this always accessed by the same thread?
maybe make it an `AtomicBoolean` instead?